src/SFA.DAS.EmployerFinance.UnitTests/SFA.DAS.EmployerFinance.UnitTests.csproj

```diff
@@ -1,13 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+
+    <IsPackable>false</IsPackable>
```

Comment:
Is this element needed?

Reply:
Yes as it stops the unit tests being packed as nuget packages. Seems from github comments about it, this is expected behaviour for unit tests. Guessing because they are class libraries they have a default of true (but I've not found any articles about it needing to be false other than the githib comments). For the web projects its false by default.